docs/bugpattern/BanSerializableRead.md

```diff
@@ -0,0 +1,12 @@
+"The Java `Serializable` API is very powerful, and very dangerous. Any
+consumption of a" serialized object that cannot be explicitly trusted will
+likely result in a critical remote code execution bug that will give an attacker
+control of the application. (See go/java-practices/serializable and
```

Comment:
The Google-internal `go/java-practices/serializable` reference may confuse readers.